src/components/Header/Header.tsx

```diff
@@ -33,17 +15,17 @@ const Header = () => {
         <div className="navbar-left">
```

Comment:
Check out the code written by Ishan, discuss nav bar's code with him and see how this component can be improved.